From: Tomáš Chvátal <tchvatal@suse.com>
Date: Thu, 25 Feb 2016 16:16:19 +0000 (+0100)
Subject: Add std prefix to isnan check to fix build with glibc > 2.23
X-Git-Url: https://gerrit.libreoffice.org/gitweb?p=core.git;a=commitdiff_plain;h=d7c64d6e430e9640c77c846006a13bbc78704c28

Add std prefix to isnan check to fix build with glibc > 2.23

Change-Id: I6321cd905152231bb013e25f7b268f85453fe917
---

diff --git a/xmloff/source/draw/ximp3dscene.cxx b/xmloff/source/draw/ximp3dscene.cxx
index 5cf205f..525e2fb 100644
--- a/xmloff/source/draw/ximp3dscene.cxx
+++ b/xmloff/source/draw/ximp3dscene.cxx
@@ -66,7 +66,7 @@ SdXML3DLightContext::SdXML3DLightContext(
             {
                 ::basegfx::B3DVector aVal;
                 SvXMLUnitConverter::convertB3DVector(aVal, sValue);
-                if (!isnan(aVal.getX()) && !isnan(aVal.getY()) && !isnan(aVal.getZ()))
+                if (!std::isnan(aVal.getX()) && !std::isnan(aVal.getY()) && !std::isnan(aVal.getZ()))
                 {
                     maDirection = aVal;
                 }
